devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy

```diff
@@ -119,6 +119,15 @@ private Map parseCatchAllMessage(String description) {
 		log.debug "parseCatchAllMessage: msgStatus: ${msgStatus}"
 		if (msgStatus == 0) {
 			switch(cluster.clusterId) {
+				case 0x0500:
+                	Map descMap = zigbee.parseDescriptionAsMap(description)
+					// someone who understands Zigbee better than me should refactor this whole DTH to bring it up to date
```

Comment:
We might even be able to combine these into our smartsense DTHs, but that's work for another day.